org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/smap/StratumBreakpointAdapterFactory.java

```diff
@@ -159,7 +161,8 @@ public Data exec(XtextResource state) throws Exception {
 			if (data.classHandle != null)
 				attributes.put(ORG_ECLIPSE_XTEXT_XBASE_CLASS_HANDLE, data.classHandle);
 
-			final IJavaStratumLineBreakpoint breakpoint = new XbaseLineBreakpoint(breakpointResource, shortName, data.types, line, charStart, charEnd, attributes);
+			
+			final IJavaStratumLineBreakpoint breakpoint = new JavaStratumLineBreakpoint(breakpointResource, shortName, null, null, data.types, line, charStart, charEnd, 0, true, attributes);
```

Comment:
Use `JDIDebugModel.createStratumBreakpoint` instead to avoid internal API usage.